microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -17,41 +17,33 @@ Publishing your Progressive Web App (PWA) to the [Microsoft Store](/windows/uwp/
+To submit your app to the Microsoft Store, you use [Microsoft Partner Center](https://partner.microsoft.com/dashboard/windows/overview).
 
 To create an app reservation:
 
-1.  To display your enrolled programs:
-    1.  Sign into Windows Partner Center with your Microsoft account and go to the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home).
-    1.  Navigate to **Windows & Xbox**.
-        *   If **Windows & Xbox** is displayed, your app is already enrolled.
-        *   If **Windows & Xbox** isn't displayed, select **Add program**.
+1.  Before you can create an app reservation, you must have a **personal** Microsoft account (not a work Microsoft account or a school Microsoft account) that is enrolled in the Windows Developer Program.
+    1.  If you don't already have one, create a personal Microsoft account at [account.microsoft.com](https://account.microsoft.com/).
+    1.  Enroll in the Windows Developer program, by following the steps in [Open a developer account in Partner Center](/windows/apps/publish/partner-center/open-a-developer-account).
 
-    ![Add a program in the Windows Partner Center dashboard](../media/windows-partner-center-add-program.msft.png)
+1.  Once your account is enrolled in the app developer program, follow these steps to create an app reservation:
+    1.  Open up the [Partner Center Dashboard](https://partner.microsoft.com/dashboard/home). Sign in with the account you enrolled.
+    1.  Under **Workspaces**, select **Apps and games**.  The **Apps and games | Overview** page appears.
+    1.  Select **New product** > **MSIX or PWA app**.
+    1.  When prompted, enter a name for your reservation, and then select **Reserve product name**.
 
-1.  To enroll in the developer program:
-    1.  Navigate to **Windows & Xbox**.
-    1.  Select **Get started**.
-    1.  Follow the prompts.
-1.  Now, your account is enrolled in the app developer program. To create an app reservation:
-    1.  Navigate to **Windows & Xbox**.
-    1.  Select **Overview** > **Create a new app**.
-    1.  Type the name of your app in the prompt.
-    1.  Select `Reserve product name`.
-
-    ![Create an app reservation in Windows Partner Center](../media/windows-partner-center-create-app.msft.png)
+    ![Create an app reservation on Microsoft Partner Center](../media/windows-partner-center-new-product.png)
 
 1.  To display your publisher details for use in the [Package your PWA](#package-your-pwa-for-the-store) section, select **Product management** > **Product Identity**.
 
-    ![Copy your publisher information from Windows Partner Center](../media/windows-partner-center-publisher-info.msft.png)
+    ![Copy your publisher information from Microsoft Partner Center](../media/windows-partner-center-publisher-info.msft.png)
     <!-- lightbox justified because large detailed image -->
 
 1.  Copy and save the following values.
```

Comment:
line ~52, change from:
```
```
to:
```
* **Publisher display name**
```
to match the UI shown in gh rendered section:
https://github.com/zateutsch/edge-developer/blob/2482-updated-pwa-partner-center-guidance/microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md#package-your-pwa-for-the-store

Reply:
done